test/test-applications/regression/MismatchedTracerVersions/Build-Tracer.ps1

```diff
@@ -0,0 +1,44 @@
+Param (
+    [String[]]
+    $TracerVersions = ('1.30.0', '1.31.0', '2.0.0'),
+
+    [ValidateScript( { Test-Path $_ -PathType 'Container' })]
+    [String]
+    $SolutionDirectory = (Join-Path $PSScriptRoot '../../../..'),
+
+    [String]
+    $DestinationDirectory = (Join-Path $PSScriptRoot 'tracer-homes')
```

Comment:
You adding more script files faster than I can replace them with nuke 😛 
![source](https://user-images.githubusercontent.com/18755388/127337387-09fb5d0c-3a75-42df-8b96-0155b91e77c0.gif)